appinfo/info.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<info>
+    <id>security</id>
+    <name>Security</name>

```

Comment:
Basically I think for community apps it's the author's right/responsibility to provide these information. If not applicable for some reason I'm fine proposing summary and description but I would need to try the app first, of course. Just tell me!